composer-require-check.json

```diff
@@ -16,6 +16,9 @@
         "void",
         "object",
         "mixed",
+        "random_bytes",
+        "events",
+        "processRequest",
```

Comment:
Those are double present now, see below.

Reply:
Oops, automerge duplicated that since we put them in different places. Fixed.